sql/core/src/test/resources/sql-tests/results/show_columns.sql.out

```diff
@@ -129,26 +129,26 @@ SHOW COLUMNS IN showdb.showcolumn3
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.catalyst.analysis.NoSuchTableException
-Table or view 'showcolumn3' not found in database 'showdb';
+org.apache.spark.sql.AnalysisException
+Table or view not found: showdb.showcolumn3; line 1 pos 0
 
 
 -- !query
 SHOW COLUMNS IN showcolumn3 FROM showdb
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.catalyst.analysis.NoSuchTableException
-Table or view 'showcolumn3' not found in database 'showdb';
+org.apache.spark.sql.AnalysisException
+Table or view not found: showdb.showcolumn3; line 1 pos 0
 
 
 -- !query
 SHOW COLUMNS IN showcolumn4
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.catalyst.analysis.NoSuchTableException
-Table or view 'showcolumn4' not found in database 'showdb';
```

Comment:
Note that in the current framework, we don't display the current db/namespace, so this database info is not available.